packages/typespec-ts/test/modularUnit/scenarios/modelsGenerator/modelsGenerator.md

```diff
@@ -740,33 +734,32 @@ export interface Foo {
 ```ts operations
 import { TestingContext as Client } from "./index.js";
 import { Foo, fooSerializer, fooDeserializer } from "../models/models.js";
+import { ReadOptionalParams } from "./options.js";
 import {
   StreamableMethod,
   PathUncheckedResponse,
   createRestError,
-  operationOptionsToRequestParameters,
+  operationOptionsToRequestParameters
 } from "@azure-rest/core-client";
 
 export function _readSend(
   context: Client,
   body: Foo,
-  options: ReadOptionalParams = { requestOptions: {} },
+  options: ReadOptionalParams = { requestOptions: {} }
 ): StreamableMethod {
-  return context
-    .path("/")
-    .post({
-      ...operationOptionsToRequestParameters(options),
-      contentType: "application/json",
-      headers: {
-        accept: "application/json",
-        ...options.requestOptions?.headers,
-      },
-      body: fooSerializer(body),
-    });
+  return context.path("/").post({
```

Comment:
Could we create an issue to track the un-necessary diffs introduced by format? It would reduce the misleadings in review.

Reply:
issue tracked https://github.com/Azure/autorest.typescript/issues/3123